doc/whats-new.rst

```diff
@@ -106,6 +106,13 @@ Deprecations
 
 Enhancements
 ~~~~~~~~~~~~
+
+- Added the xarray equivalent of `pandas.Dataframe.combine_first` as an instance
+  method to DataArray/Dataset objects, facilitated by the new `ops.fillna` that
+  uses `apply_ufunc` for different `join` options.
```

Comment:
I would drop implementation details like the fact that this uses `apply_ufunc` from the user facing release notes.